zerolog/zerolog.go

```diff
@@ -0,0 +1,70 @@
+package zerolog
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/airbrake/gobrake/v5"
+	"github.com/buger/jsonparser"
+	"github.com/rs/zerolog"
+)
+
+type Writer struct {
+	Gobrake *gobrake.Notifier
+}
+
+// New creates a new Writer
+func New(notifier *gobrake.Notifier) *Writer {
+	return &Writer{Gobrake: notifier}
```

Comment:
Should this panic if `notifier` is nil to prevent errors referencing `.Gobrake` later?

Reply:
I dont think we should panic, (3rd party packages should not panic, but should send info back up so the user knows that they need to handle it or panic themselves) 

I can add a check to see if it's defined, and if not return an error stating airbrake must be set.

Reply:
Sounds good